pkg/reader/helpers.go

```diff
@@ -69,12 +73,17 @@ func bindReaderContract(
 		Name:    contractName,
 	}
 
+	lggr.Debugw("Binding contract",
+		"chainSel", chainSel,
+		"contractName", contractName,
+		"address", encAddress,
```

Comment:
Just a note, but this doesn't indicate if its a no-op or not, you'd need to analyze the logs to see if chainSel/contractName/address combo has not already been used.